docs/src/main/sphinx/admin/dynamic-filtering.rst

```diff
@@ -55,10 +64,10 @@ Dynamic filtering depends on a number of factors:
 * Connector support for utilizing dynamic filters at the splits enumeration stage.
 * Size of right (build) side of the join.
 
-You can take a closer look at the :doc:`EXPLAIN plan </sql/explain>` of the query
-to analyze if the planner is adding dynamic filters to a specific query's plan.
-For example, the explain plan for the above query can be obtained by running
-the following statement::
+You can take a closer look at the :doc:`EXPLAIN plan </sql/explain>` of the
+query to analyze if the planner is adding dynamic filters to a specific query's
```

Comment:
"The query's doc:`EXPLAIN plan </sql/explain>` allows you to to analyze..." - gets rid of the "can" and the longer possessive form.